cfme/tests/cloud_infra_common/test_power_control_rest.py

```diff
@@ -134,6 +136,7 @@ def test_start_vm_rest(appliance, vm_obj, verify_vm_stopped, soft_assert, from_d
     soft_assert(verify_vm_power_state(vm, vm_obj.STATE_ON), "vm not running")
 
 
+@pytest.mark.rhv2
```

Comment:
I do think suspend VM action should be included in tier 2. it is a basic action that should work. If there is a bug in this I think it will be better to find it in tier 2.